.github/workflows/ci.yml

```diff
@@ -64,6 +72,13 @@ jobs:
         env:
           SKIP_LINT: 1
 
+      - name: Apply Calico Rules to LKE
+        if: always()
+        run: |
+          cd scripts && ./lke_calico_rules_e2e.sh
+        env:
+          LINODE_TOKEN: ${{ secrets.DX_LINODE_TOKEN }}
+
```

Comment:
I thin k the CI never hit the API because it's mocking everything. Is it required for some cases I missed?

Reply:
Good catch just reverted the ci.yml